backends/openvino/README.md

```diff
@@ -0,0 +1,88 @@
+# OpenVINO Backend for ExecuTorch
+The OpenVINO backend enables optimized execution of deep learning models on Intel hardware, leveraging Intel's [OpenVINO toolkit](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) for inference acceleration.
+
+## Supported Hardware
+
+OpenVINO backend supports the following hardware:
+
+- Intel CPUs
+- Intel integrated GPUs
+- Intel discrete GPUs
+- Intel NPUs
+
+## Directory Structure
+
+```
+executorch
+├── backends
+│   └── openvino
+│       ├── runtime
+│           ├── OpenvinoBackend.cpp
+│           └── OpenvinoBackend.hpp
+│       ├── scripts
+│           └── openvino_build.sh
+│       ├── tests
+│       ├── CMakeLists.txt
+│       ├── README.md
+│       ├── __init__.py
+│       ├── openvino_functions.yaml
+│       ├── partitioner.py
+│       ├── preprocess.py
+│       └── requirements.txt
+└── examples
+│   └── openvino
+│       ├── aot
+│           ├── README.md
+│           └── aot_openvino_compiler.py
+│       └── executor_runner
+│           └── openvino_executor_runner.cpp
+│       ├── CMakeLists.txt
+│       ├── README.md
+└──     └── openvino_build_example.sh
+```
+
+## Build Instructions
+
+### Prerequisites
+
+Before you begin, ensure you have openvino installed and configured on your system:
```

Comment:
- Can you link to any page or instructions that tell you how to do this?
- I hope there is a clear error/info message when user tries to use openvino backend but pre-reqs are not installed.
- Is this required both for AOT and runtime?

Reply:
- For the instructions, the steps below the corresponding sentence should be explaining how to build and install openvino. Please let us know if further improvement is needed on this instructions.
- If the pre-reqs are missing, error messages will be shown as the required modules/libraries will not be found.
- Yes, both AOT and runtime requires openvino to be installed.